oaknational/Oak-Web-Application
# Fix/download hubspot form

## Description

add posthog user id to form
handle no school correctly
returns completed form with all AC data

## Issue(s)

Fixes #1577 

## How to test

1. Go to downloads page
2. Click on download
3. You should see forms being filled in on hubspot

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
